--- magicor-1.1/Makefile~	2008-07-08 10:37:17.000000000 +0200
+++ magicor-1.1/Makefile	2008-11-21 16:43:20.000000000 +0100
@@ -15,15 +15,15 @@
 #PYTHON_VERSION=2.4
 
 # The prefix is added as a prefix (duh) to all other paths as default.
-PREFIX=/usr/local
+PREFIX=$(DESTDIR)/usr
 # Where the magicor package will be installed, including final directory.
-PYTHON_LIB=$(PREFIX)/lib/python$(PYTHON_VERSION)/site-packages/magicor
+PYTHON_LIB=$(PREFIX)/share/python$(PYTHON_VERSION)/site-packages/magicor
 # Where the executable scripts will be installed.
-BIN_PATH=$(PREFIX)/games
+BIN_PATH=$(PREFIX)/bin
 # Where the shared data will be stored.
 SHARE_PATH=$(PREFIX)/share/magicor
 # Where to find the system-wide config file.
-CONFIG=$(PREFIX)/etc
+CONFIG=$(DESTDIR)/etc
 # Docbook XSL stylesheet to use
 DOCBOOK_STYLESHEET=/usr/share/xml/docbook/stylesheet/nwalsh/xhtml/docbook.xsl
 # Options (string parameters) passed to the XSLT processor
@@ -62,7 +62,7 @@
 	chmod a+x $(BIN_PATH)/magicor
 	chmod a+x $(BIN_PATH)/magicor-editor
 	mkdir -p $(SHARE_PATH)
-	cp -fr data/* $(SHARE_PATH)
+	cp -fr  $(MAGICOR)/data/* $(SHARE_PATH)
 	@echo "Done. If everything wen't well you can now run '$(BIN_PATH)/Magicor'"
 
 .PHONY: uninstall
