--- magicor-1.0-rc2/Magicor.py~	2006-11-20 02:07:17.000000000 +0100
+++ magicor-1.0-rc2/Magicor.py	2006-11-25 12:09:47.000000000 +0100
@@ -28,8 +28,8 @@
     baseConf=".\magicor.conf"
 else:
     parser.add_option("-c", "--config", dest="configPath",
-                  default = "###CONFIG_PATH###",
-                  help="use this default config, default ###CONFIG_PATH###")
+                  default = "/etc/magicor.conf",
+                  help="use this default config, default /etc/magicor.conf")
     baseConf="~/.magicor/magicor.conf"
 
 parser.add_option("-j", "--joystick",
--- magicor-1.0-rc2/etc/magicor.conf~	2006-11-18 12:09:21.000000000 +0100
+++ magicor-1.0-rc2/etc/magicor.conf	2006-11-25 12:15:27.000000000 +0100
@@ -1,4 +1,4 @@
-data_path = ###SHARE_PATH###
+data_path = /usr/share/magicor
 #fullscreen = 0
 #music = 1
 #sound = 1
--- magicor-1.0-rc2/etc/magicor-editor.conf~	2006-11-18 12:12:06.000000000 +0100
+++ magicor-1.0-rc2/etc/magicor-editor.conf	2006-11-25 12:18:01.000000000 +0100
@@ -1,2 +1,2 @@
-data_path = ###SHARE_PATH###
+data_path = /usr/share/magicor
 remember_windows = yes
--- magicor-1.0-rc2/Magicor-LevelEditor.py~	2006-11-20 02:07:17.000000000 +0100
+++ magicor-1.0-rc2/Magicor-LevelEditor.py	2006-11-25 12:38:35.000000000 +0100
@@ -30,8 +30,8 @@
     baseConf=".\magicor-editor.conf"
 else:
     parser.add_option("-c", "--config", dest="configPath",
-                  default = "###CONFIG_PATH###",
-                  help="use this default config, default ###CONFIG_PATH###")
+                  default = "/etc/magicor.conf",
+                  help="use this default config, default /etc/magicor.conf")
     baseConf="~/.magicor/magicor-editor.conf"
 ##<--win
 
